lib/core/src/examples/shapes.rs

```diff
@@ -24,6 +26,24 @@ pub fn run_example_shapes() {
 }
 
 fn init(world: &World) {
+    let container      = DomContainer::from_id("app").expect("Couldn't get container");
+    let mut event_loop = world.event_loop();
+    let mut camera     = None;
+    world.scene(|scene| camera = Some(scene.camera()));
+    let camera     = camera.unwrap();
+    camera.update();
+
+    let screen = camera.screen();
+    let fovy_slope = camera.half_fovy_slope();
+    let x = 0.0;
+    let y = 0.0;
+    let z = screen.height / 2.0 / fovy_slope;
+    camera.set_position(Vector3::new(x, y, z));
```

Comment:
This file should show how users of basegl are meant to use the library. Requiring the user to write such boileprlate code is a bad design. This code should not be here. The code should be as simple as the threejs API. Moreover, attaching a navigator should NOT move the camera. Attaching navigator should keep camera where it was. The whole code should look like this here:

```
fn init(world: &World) {
    let camera = world.scene().camera();
    let navigator = Navigator::new(&camera, &scene).ok();

    let s1 = Circle("25.0 + 20.0*sin(input_time/1000.0)");
    ...
```

Reply:
@wdanilo wait, but this is the initialization of the camera position for the example. We have a different initialization for `camera_navigation.rs`.

We can of course define the initial position of the camera. It can be initialized to `vec3(width/2,height/2,height/2/fovy_slope)` in `Scene::new`

Reply:
Camera was always initialized automatically in these examples, why we need to initialize it by hand now?